eras/shelley/impl/src/Cardano/Ledger/Shelley/PParams.hs

```diff
@@ -434,13 +437,14 @@ shelleyPParamsUpdatePairs pp =
 
 shelleyPParamsHKDPairs ::
   forall f era.
-  (HKDFunctor f, EraPParams era, ProtVerAtMost era 4, ProtVerAtMost era 6) =>
+  (HKDFunctor f, EraPParams era, ProtVerAtMost era 4, ProtVerAtMost era 6, ProtVerAtMost era 8) =>
   Proxy f ->
   PParamsHKD f era ->
   [(Key, HKD f Aeson.Value)]
 shelleyPParamsHKDPairs px pp =
   shelleyCommonPParamsHKDPairs px pp
     ++ shelleyCommonPParamsHKDPairsV6 px pp
+    ++ shelleyCommonPParamsHKDPairsV8 px pp
```

Comment:
You need to add this also to Alonzo PParams here:
https://github.com/input-output-hk/cardano-ledger/blob/48d661803c40306dc8f58ce77ab077aecf873338/eras/alonzo/impl/src/Cardano/Ledger/Alonzo/PParams.hs#L627-L630